contrib/ansible/Getting-Started.md

```diff
@@ -0,0 +1,157 @@
+
+
+# Kubernetes Cluster with Containerd and Cri-Containerd
+
+In this document we aim to provide simple steps to bring up a Kubernetes cluster using ansible and kubeadm tools. 
```

Comment:
This document provides the steps to bring up a Kubernetes cluster using ansible and kubeadm.